v2/internal/frontend/desktop/windows/frontend.go

```diff
@@ -499,7 +511,7 @@ func (f *Frontend) setupChromium() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	err = settings.PutAreDefaultContextMenusEnabled(f.devtools || f.frontendOptions.EnableDefaultContextMenu)
+	err = settings.PutAreDefaultContextMenusEnabled(f.frontendOptions.EnableDefaultContextMenu)
```

Comment:
This change disables the default context-menu on windows in dev & debug because  `f.devtools` is enabled for `-devtools`, `-debug` and dev mode.